dataplane/src/packet_hash.rs

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+//
+//! Module to compute packet hashes
+
+use super::Packet;
+use net::buffer::PacketBufferMut;
+use net::headers::{TryHeaders, TryTransport};
+
+use ahash::AHasher;
+use net::headers::Net;
+use net::headers::Transport;
+use net::headers::TryIp;
+use std::hash::Hash;
+use std::hash::Hasher;
```

Comment:
Formatting nit:

```suggestion
use ahash::AHasher;
use net::buffer::PacketBufferMut;
use net::headers::{Net, Transport, TryHeaders, TryIp, TryTransport};
use std::hash::{Hash, Hasher};
```

(Not tested.)

Reply:
@qmonnet I have manually cleaned up the imports since you said the above wasn't tested.
Thanks!